src/Time/TimeSteppers/TimeStepper.hpp

```diff
@@ -42,7 +43,8 @@ DEFINE_FAKE_VIRTUAL(update_u)
 /// Abstract base class for TimeSteppers.
 class TimeStepper : public PUP::able {
  public:
-  using Inherit = TimeStepper_detail::FakeVirtualInherit_update_u<TimeStepper>;
+  using Inherit = TimeStepper_detail::FakeVirtualInherit_update_u<
+      TimeStepper_detail::FakeVirtualInherit_update_u<TimeStepper>>;
```

Comment:
maybe I'm confused, but I expected one of these to be `FakeVirtualInherit_dense_output`

Reply:
I am also confused by this.  Otherwise I am happy with the PR

Reply:
Yeah, that was wrong.